meta-mender-core/recipes-mender/mender/mender.inc

```diff
@@ -10,7 +10,7 @@ def cert_location_if_server_crt_in(src_uri, d):
     return ""
 
 MENDER_SERVER_URL ?= "https://docker.mender.io"
-MENDER_CERT_LOCATION ?= "${@cert_location_if_server_crt_in('${SRC_URI}', d)}"
+MENDER_CERT_LOCATION ??= "${@cert_location_if_server_crt_in('${SRC_URI}', d)}"
```

Comment:
For my own education, but why do we change it to weak assignment?

Reply:
Because of the other weak assignment I introduced in the same commit. We want it to override this one, but we don't want it to override the users' assignments using `=`.